assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+# Notary Project Docs Assessment
+
+Prepared by: `Nate Waddington`
+Date: `2021-07-07`
+Project: [Notary Project](https://github.com/notaryproject/)
+
+## Introduction
+
+This document assesses the quality and completeness of a project's documentation and website (if present).
+
+This document:
+
+- Measures existing documentation quality against the CNCF’s standards
+- Recommends specific and general improvements
+- Provides examples of great documentation as reference 
+- Identifies key improvements with the largest return on investment
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+## Project documentation 
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required) |
+
+### V1
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           |     |     | ✅  |     |     |
+| Content maintainability    |     | ✅  |     |     |     |
+| Content creation processes |     |     | ✅  |     |     |
+
+### V2
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           | ✅  |     |     |     |     |
+| Content maintainability    | ✅  |     |     |     |     |
+| Content creation processes | ✅  |     |     |     |     |
+
+### Comments
+
+There are two versions of notary that need to be supported with documentation.
+- V1 is housed with [The Update Framework](https://github.com/theupdateframework/notary), and the docs are included as a set of markdown files in the [/docs folder](https://github.com/theupdateframework/notary/tree/master/docs)
+- V2 is housed with The Notary Project (https://github.com/notaryproject/notaryproject) 
```

Comment:
Are there two versions of documentation or are there two documentation repositories? 

From what I can see, there are two documentation repositories, and their v2 docs are currently spec/scenario/use case documentation rather than end user documentation.

A recommendation I would make to this project is that they clearly understand that use case documentation is irrelevant to their end user, and that they need to provide documentation for those end users as well.

Reply:
> Are there two versions of documentation or are there two documentation repositories?

There are 2 versions of Notary, and V2 started new (seemingly) from the ground up. So the documentation versions won't necessarily flow one from the other.
 
> From what I can see, there are two documentation repositories, and their v2 docs are currently spec/scenario/use case documentation rather than end user documentation.

This is correct, V2 docs are primarily specifications. There is a [V2 cli prototype](https://github.com/notaryproject/nv2) with some documentation as well.